src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -507,6 +509,22 @@ struct amdxdna_drm_query_firmware_version {
 	__u32 build; /* out */
 };
 
+/**
+ * struct amdxdna_drm_query_telemetry - Query preemption counter via telemetry
+ * @major: The major version number
+ * @minor: The minor version number
+ * @reserved: Reserved
+ * @layer_boundary_count: Layer boundary preemption counts indexed with user ID
+ * @frame_boundary_count: frame boundary preemption counts indexed with user ID
+ */
+struct amdxdna_drm_query_telemetry {
+	__u32 major;
+	__u32 minor;
+	__u64 reserved[352];
+	__u64 layer_boundary_count[16];
+	__u64 frame_boundary_count[16];
+};
+
```

Comment:
I don't see "struct amdxdna_drm_query_telemetry" is used in the driver. Please move this to device.cpp.